owlbot.py

```diff
@@ -86,15 +85,15 @@ def get_staging_dirs(
     s.move(library / "scripts")
 
 for library in get_staging_dirs(bigtable_admin_default_version, "bigtable_admin"):
-    is_fresh_admin_copy = \
-        s.move(library / "google/cloud/bigtable_admin", excludes=["**/gapic_version.py"])
     is_fresh_admin_v2_copy = \
-        s.move(library / "google/cloud/bigtable_admin_v2", excludes=["**/gapic_version.py"])
-    s.move(library / "tests")
+        s.move(library / "google/cloud/bigtable/admin_v2",
```

Comment:
Do we need a CL to change `google/cloud/bigtable_admin_v2` to `google/cloud/bigtable/admin_v2`? 

Currently we generate `google/cloud/bigtable_admin_v2`

https://github.com/googleapis/googleapis-gen/tree/master/google/bigtable/admin/v2/bigtable-admin-v2-py/google/cloud/bigtable_admin_v2

If we want `google/cloud/bigtable/admin_v2` then we'll need to delete these lines
https://github.com/googleapis/googleapis/blob/a17b84add8318f780fcc8a027815d5fee644b9f7/google/bigtable/admin/v2/BUILD.bazel#L173-L174